app/service/default_group_service.rb

```diff
@@ -0,0 +1,45 @@
+class DefaultGroupService
+  def create_trial_user_default_group!(user)
+    return unless user.trial? && user.name.present? && user.organisation.present?
+
+    forms = Form.where(creator_id: user.id).to_h { [_1.id, _1] }
+    form_ids = forms.keys
+    group_form_ids = GroupForm.where(form_id: form_ids).pluck(:form_id)
+    not_group_form_ids = form_ids.to_set - group_form_ids
+
+    if not_group_form_ids.blank?
+      Rails.logger.info "default_groups: skipping trial user #{user.name}"
+      return
+    end
+
+    Rails.logger.info "default_groups: trial user #{user.name}"
+
+    default_trial_group = Group.find_or_create_by!(
+      creator_id: user.id,
+      name: "#{user.name}’s trial group",
+      organisation_id: user.organisation_id,
+      status: :trial,
+    ) do |new_group|
+      Rails.logger.info "default_groups: created #{new_group.name}, with creator #{new_group.creator.email}"
```

Comment:
Is there a way we can avoid logging user names and email addresses while still logging something useful? I'm not sure the exact rules around logging PII for our internal users but it feels safer not to unless we absolutely need to.

Could we log the user `id` and/or `uid` instead?

Reply:
We already log email address for all users in the admin app

Reply:
Ah ok, if this isn't something new and we've decided it's ok then don't see an issue.